### ISSUE TITLE
Global variable warning only appears if there's a norm error

### ISSUE THREAD

**Issue by ScarletPixie** [42School/norminette#501]
The norminette program only shows the global variable detected notice if there's some norm error in the file.

code:
```c
//NO warning:

char const	*g_map[] = {
	"111111",
	"  1001",
	"111P01",
	"100001",
	"11W11",
	"11111",
	NULL,
};

int	main(void)
{
	return (0);
}
```

```c
//GIVES a both a norm error (no new line after var declaration and the global variable warning):
char const	*g_map[] = {
	"111111",
	"  1001",
	"111P01",
	"100001",
	"11W11",
	"11111",
	NULL,
};

int	main(void)
{
	char	*something;
	(void)something;
	return (0);
}
```

**Additional infos**
norminette 3.3.54

**Additional context**
"no warning"
![Screenshot from 2024-06-20 10-16-23](https://github.com/42School/norminette/assets/115340289/87806391-2b8c-4c2f-8531-774e6474a8e3)

"error and warning"
![Screenshot from 2024-06-20 10-17-37](https://github.com/42School/norminette/assets/115340289/55896399-cff9-419c-b6a3-5e99f8951661)

**Comment by NiumXp** [comment]
Hello, @ScarletPixie!

This issue is already fixed in norminette 3.3.55 according to https://github.com/42School/norminette/pull/459.

**Comment by ScarletPixie** [comment]
got it

**Comment by ScarletPixie** [comment]
fixed on version 3.3.55
